build.gradle

```diff
@@ -1,119 +1,35 @@
-
-import java.time.Instant
-import java.time.LocalDate
-
 plugins {
-    id 'java'
+    id("uk.gov.justice.hmpps.gradle-spring-boot") version "0.3.0"
     id 'groovy'
-    id 'org.springframework.boot' version '2.2.7.RELEASE'
-    id 'io.spring.dependency-management' version '1.0.9.RELEASE'
-    id "com.gorylenko.gradle-git-properties" version "2.2.2"
-    id 'com.github.ben-manes.versions' version '0.28.0'
-    id 'org.owasp.dependencycheck' version '5.3.2.1'
-    id 'se.patrikerdes.use-latest-versions' version '0.2.13'
-}
-
-repositories {
-    mavenLocal()
-    mavenCentral()
-}
-
-def isNonStable = { String version ->
-    def stableKeyword = ['RELEASE', 'FINAL', 'GA'].any { it -> version.toUpperCase().contains(it) }
-    def regex = /^[0-9,.v-]+(-r)?$/
-    return !stableKeyword && !(version ==~ regex)
-}
-
-dependencyUpdates {
-    rejectVersionIf {
-        isNonStable(it.candidate.version) && !isNonStable(it.currentVersion)
-    }
-}
-
-configurations {
-    agentDeps
-    agentDeps.transitive = false
 }
 
-dependencyCheck {
-  failBuildOnCVSS = 5
-  suppressionFiles = ['dependency-check-suppress-spring.xml']
-  format = 'ALL'
-  analyzers {
-    assemblyEnabled = false
-  }
-}
-project.sourceCompatibility = 11
-project.targetCompatibility = 11
-
 group = 'uk.gov.justice.digital.delius'
```

Comment:
be interesting to know whether this value has priority or whether the one from the plugin trumps it?

Reply:
My understanding is that Gradle first applies the plugins then applies the buildscript - so I think it should override with this one :crossed_fingers: